config/routes.rb

```diff
@@ -40,4 +42,8 @@
     concerns :gbl_downloadable
   end
   resources :download, only: [:show]
+
+  # Map OkComputer's /health/all.json to /health
+  # Don't mount any other routes
+  get "/health", to: "ok_computer/ok_computer#index", defaults: { format: :json }
```

Comment:
All our other repos use single quotes instead of double quotes (as does this file above, i.e. L14, L19).

```suggestion
  get '/health', to: 'ok_computer/ok_computer#index', defaults: { format: :json }
```